infra/cluster-init/init.sh

```diff
@@ -47,14 +47,14 @@ function install_knative {
     kubectl apply \
         -f "https://github.com/knative/serving/releases/download/knative-v${KNATIVE_VERSION:-1.0.1}/serving-hpa.yaml"
 
-    kubectl apply \
-        -f knative-configmaps/config-features.yaml
-
-    local core_apps=("activator" "autoscaler" "controller" "webhook")
+    local core_apps=("activator" "autoscaler" "controller" "webhook" "domain-mapping" "domainmapping-webhook")
```

Comment:
just for my understanding what this addition for? `"domain-mapping" "domainmapping-webhook"`

Reply:
This is additional component that knative introduces recently. Adding these 2 for completeness to ensure that knative is completely ready.